pipeline/inputs/node-exporter-metrics.md

```diff
@@ -15,13 +15,28 @@ The initial release of Node Exporter Metrics contains a subset of collectors and
 This plugin is currently only supported on Linux based operating systems\
 
 
-## Configuration 
+## Configuration
 
 | Key             | Description                                                            | Default   |
 | --------------- | ---------------------------------------------------------------------- | --------- |
 | scrape_interval | The rate at which metrics are collected from the host operating system | 5 seconds |
 | path.procfs     | The mount point used to collect process information and metrics        | /proc/    |
 | path.sysfs      | The path in the filesystem used to collect system metrics              | /sys/     |
+| collector.cpu.scrape\_interval       | The rate at which cpu metrics are collected from the host operating system. To enable for a specific collection interval, specify greater than 0. | 0 seconds |
```

Comment:
Something like this for each of them?
```suggestion
| collector.cpu.scrape\_interval | The rate in seconds at which cpu metrics are collected from the host operating system. If a value greater than 0 is used then it overrides the global default otherwise the global default is used. | 0 seconds |
```